src-tauri/src/main.rs

```diff
@@ -628,3 +631,30 @@ fn launch_northstar_steam(
 
     retval
 }
+
+#[tauri::command]
+async fn install_northstar_proton_wrapper() -> Result<(), String> {
+    #[cfg(target_os = "linux")]
+    return linux::install_ns_proton().map_err(|err| err.to_string());
+
+    #[cfg(target_os = "windows")]
+    Err("Not supported on Windows".to_string())
+}
+
+#[tauri::command]
+async fn uninstall_northstar_proton_wrapper() -> Result<(), String> {
+    #[cfg(target_os = "linux")]
+    return linux::uninstall_ns_proton();
+
+    #[cfg(target_os = "windows")]
+    Err("Not supported on Windows".to_string())
+}
+
+#[tauri::command]
+fn get_local_northstar_proton_wrapper_version() -> Result<String, String> {
```

Comment:
Why is this one not `async`? ^^

Reply:
The function is so short and simple that I think its not worth spinning up a thread for it.
It would only really run into problems, if something was completely exhausting the Disk IO, but even then it would be a drop of water in an ocean.